tests/glsl-intrinsic/shader-subgroup/shader-subgroup-shuffle.slang

```diff
@@ -0,0 +1,139 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv -emit-spirv-directly
+
+// not testing hlsl due to missing impl
+//DISABLE_TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl -DTARGET_HLSL
+// not testing cuda due to missing impl
+//DISABLE_TEST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda -DTARGET_CUDA 
+// not testing cpp due to missing impl
+//DISABLE_TEST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp -DTARGET_CPP
+
+//TEST(compute, vulkan):COMPARE_COMPUTE(filecheck-buffer=BUF):-vk -compute -entry computeMain -allow-glsl
```

Comment:
These runnable tests seem to have inconsistency on the CI server across runs. We either need to get to the bottom of that, or simply disable runnable tests and rely on more thorough SIMPLE test pattern checking.

Reply:
I will look into the CI consistency issue (I believe it is likely related to subgroup size & partitioning)

Reply:
looking at the test logs more and the probably launch order of tests, arithmetic.5 & arithmetic.6 on full capability tests finish quite a bit after the other arithmetic tests when pass/fail; if they run only a partial test they finish very close to the other arithmetic tests; I believe this is a sign that the tests were taking too long due to testing too many types & permutations 

(likely need to split up the test)

Reply:
Yeah, it could be a timeout issue.